aptos-move/framework/aptos-stdlib/sources/copyable_any.spec.move

```diff
@@ -4,7 +4,7 @@ spec aptos_std::copyable_any {
     // Function specifications
     // -----------------------
 
-    spec pack<T: drop + store>(x: T): Any {
+    spec pack<T: drop + store + copy>(x: T): Any {
```

Comment:
Why did this change happen in this PR?

Reply:
This is because abilities were wrongly implemented before. You could list here whatever abilities you want before instead of those the original fun decl had. The refactor fixed this (kind of unintentional) and so I needed to also correct the source here.

Reply:
I suspected as much, could you add it to the description which bugs were fixed?